### PR TITLE
Dark mode: Fix expando arrow color.

### PR DESCRIPTION
**Pull request by nhnt11**

### CODE REVIEW

File: static/css/mozsearch.css [hunk 1]
```diff
@@ -75,9 +78,12 @@ table {
 @media (prefers-color-scheme: dark) {
   :root {
     --page-background: rgb(28, 27, 34);
-    --page-foreground: rgb(251, 251, 254);
+    --page-foreground: rgb(210, 211, 212);
```
Comment (emilio) [inline static/css/mozsearch.css:81]: As per the bug comment, these come from our in-content pages so I'd rather keep foreground and background as-is unless there's a good reason not to do that.

File: static/css/mozsearch.css [hunk 2]
```diff
@@ -127,6 +133,9 @@ body {
 h4 {
   margin: 0 0 0.5rem 0;
 }
+a {
+  color: 
+}
```
Comment (emilio) [inline static/css/mozsearch.css:138]: Please remove this?

File: static/css/mozsearch.css [hunk 3]
```diff
@@ -430,7 +439,7 @@ tr.after-context-line + tr.before-context-line {
 
 .expando {
   cursor: pointer;
-  color: black;
+  color: var(--page-foreground);
```
Comment (emilio) [inline static/css/mozsearch.css:442]: Can this just be currentColor or inherit instead?

File: static/css/mozsearch.css [hunk 4]
```diff
@@ -497,15 +506,15 @@ tr.after-context-line + tr.before-context-line {
 .bubble a:visited,
 .result-head a:link,
 .result-head a:visited {
-  color: #0095dd;
+  color: var(--link-color);
 }
 .breadcrumbs a:hover,
 .result-head a:hover {
-  color: #00539f;
+  color: var(--link-hover-color);
```
Comment (emilio) [inline static/css/mozsearch.css:513]: https://github.com/mozsearch/mozsearch/pull/468 has some additional clean-ups here, might be worth incorporating, but happy to do them in a follow-up instead.

**Review by emilio — CHANGES_REQUESTED** [review]
Thanks!

File: static/css/mozsearch.css (hunk 1 above)
Comment (nhnt11) [reply static/css/mozsearch.css:81]: Hmm, ok, that's fair. FWIW GitHub also has a lighter foreground color value similar to Bugzilla. I personally find it more pleasant but I get the design system consistency aspect.

File: static/css/mozsearch.css (hunk 3 above)
Comment (nhnt11) [reply static/css/mozsearch.css:439]: Apparently not. Inherit/currentColor gets the color from a .left-column element which also holds line numbers and is colored #aaa.

File: static/css/mozsearch.css (hunk 2 above)
Comment (nhnt11) [reply static/css/mozsearch.css]: done

File: static/css/mozsearch.css (hunk 4 above)
Comment (nhnt11) [reply static/css/mozsearch.css:510]: If you're working on this actively please feel free to even reject this and e.g. only land the expando change. I might not check back here for a while once I jump into meetings.

File: static/css/mozsearch.css (hunk 1 above)
Comment (nhnt11) [reply static/css/mozsearch.css]: reverted

File: static/css/mozsearch.css (hunk 4 above)
Comment (emilio) [reply static/css/mozsearch.css:510]: Ok, I'll land that and rebase your branch to land the expando change. That way we probably get it on the next indexer run.

**Pull request merged**